models/Property.php

```diff
@@ -25,17 +25,17 @@
  */
 final class Property extends AbstractModel
 {
-    protected string $name;
+    protected ?string $name = null;
 
     protected mixed $data = null;
 
-    protected string $type;
+    protected ?string $type = null;
 
-    protected string $ctype;
+    protected ?string $ctype = null;
 
     protected ?string $cpath = null;
 
-    protected int $cid;
+    protected ?int $cid = null;
```

Comment:
Hi @blankse, can this really be null? Based on the properties table cid, ctype and name are not null-able.

Reply:
Yes for new unsaved Properties:
```
$property = new Property();
$property->getCid();
```

Reply:
I see, thank you